src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -122,11 +127,32 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
                         isAlertVisible={!!errorMessage}
                         isDisabled={isOffline}
                         isLoading={issueNewCard?.isLoading}
-                        onSubmit={submit}
+                        onSubmit={() => setIsValidateCodeActionModalVisible(true)}
                         buttonText={translate('workspace.card.issueCard')}
                     />
                 </View>
             </ScrollView>
+            {!!issueNewCard && (
+                <ValidateCodeActionModal
+                    handleSubmitForm={submit}
+                    isLoading={issueNewCard?.isLoading}
+                    sendValidateCode={() => User.requestValidateCodeAction()}
+                    validateError={validateError}
+                    hasMagicCodeBeenSent={validateCodeSent}
```

Comment:
We aren't not resetting the value of `validateCodeSent` which causes the issue https://github.com/Expensify/App/issues/52784